### PR TITLE
Merging in new logic/aux data

### PR DESCRIPTION
**Pull request by mike8699**

### CODE REVIEW

File: ph_rando/shuffler/logic/SW Sea/Nyave Boat/NyaveBoat.logic [hunk 1]
```diff
@@ -2,4 +2,17 @@ area NyaveBoat:
...
+      enemy Geozard
+
+    node EnemiesDefeated:
+      chest Gift
+
+    node TradingSequence:
+      chest WoodHeart
+
+  Main <-> EnemiesDefeated: (defeated PirateBlin1 & defeated PirateBlin2 & defeated PirateBlin3 & defeated PirateBlin4 & defeated Geozard)
+  EnemiesDefeated <-> TradingSequence: item GuardNotebook
```
Comment (mike8699) [inline ph_rando/shuffler/logic/SW Sea/Nyave Boat/NyaveBoat.logic:18]:
```suggestion
    Main <-> EnemiesDefeated: (defeated PirateBlin1 & defeated PirateBlin2 & defeated PirateBlin3 & defeated PirateBlin4 & defeated Geozard)
    EnemiesDefeated <-> TradingSequence: item GuardNotebook
```

File: ph_rando/shuffler/logic/Dungeons/Goron Temple/GoronTemple.logic [hunk 2]
```diff
@@ -33,6 +37,8 @@ area GoronTemple:
     Arena -> AfterArena: (defeated ArmosArenaRight & defeated ArmosArenaLeft)
     AfterArena -> Entrance
     AfterArena <-> BowTarget: item Bow
+    AfterArena -> BombableWalls: item Bombs | item Bombchu
```
Comment (mike8699) [inline ph_rando/shuffler/logic/Dungeons/Goron Temple/GoronTemple.logic:40]:
```suggestion
    AfterArena -> BombableWalls: (item Bombs | item Bombchu)
```

File: ph_rando/shuffler/logic/SE Sea/Goron Island/Outside/LowerLeft.logic [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    node EnemyLedge:
+      enemy YellowChuChu1
+      enemy YellowChuChu2
+      enemy YellowChuChu3
+
+    node EnemyDefeated:
+      chest TreasureMap # SE 5
+
+    Ledge -> Port
+    Ledge <-> EnemyLedge: item Shovel & (item Bow | item Hammer)
```
Comment (mike8699) [inline ph_rando/shuffler/logic/SE Sea/Goron Island/Outside/LowerLeft.logic:25]:
```suggestion
    Ledge <-> EnemyLedge: (item Shovel & (item Bow | item Hammer))
```

File: ph_rando/shuffler/logic/SE Sea/Goron Island/Outside/LowerLeft.logic [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      enemy YellowChuChu1
+      enemy YellowChuChu2
+      enemy YellowChuChu3
+
+    node EnemyDefeated:
+      chest TreasureMap # SE 5
+
+    Ledge -> Port
+    Ledge <-> EnemyLedge: (item Shovel & (item Bow | item Hammer))
+    EnemyLedge <-> EnemyDefeated: defeated YellowChuChu1 & defeated YellowChuChu2 & defeated YellowChuChu3
```
Comment (mike8699) [inline ph_rando/shuffler/logic/SE Sea/Goron Island/Outside/LowerLeft.logic:26]:
```suggestion
    EnemyLedge <-> EnemyDefeated: (defeated YellowChuChu1 & defeated YellowChuChu2 & defeated YellowChuChu3)
```

**Comment by mike8699** [comment]
@Redriel57 I fixed the errors CI was reporting, I think this is good now.

**Review by Redriel57 — APPROVED** [review]

**Comment by mike8699** [comment]
CI is reporting that there are some Nodes that are not connected to the rest of the graph. I'm not convinced that test is even accurate, so I think it's better to merge this now and I'll take a look at that test after

**Pull request merged**